lib/internal/repl.js

```diff
@@ -84,17 +99,22 @@ function setupHistory(repl, historyPath, ready) {
     if (err) {
       return ready(err);
     }
-
     if (data) {
+      repl.history = data.split(/\n|\r/).slice(-repl.historySize);
```

Comment:
@thefourtheye see above comment from @silverwind .
The idea is to be able to share the history file between different platforms.